tests/dashbio_demos/app_onco_print.py

```diff
@@ -102,219 +102,206 @@ def header_colors():
+                id='oncoprint-tabs',
+                children=[
+                    dcc.Tab(
+                        label='About',
+                        value='what-is',
+                        children=html.Div(className='oncoprint-tab', children=[
+                            html.H4(
+                                "What is OncoPrint?"
+                            ),
+                            html.P(
+                                """
+                                The OncoPrint component is used to view multiple genetic
+                                alteration events through an interactive and zoomable
+                                heatmap. It is a React/Dash port of the popular
+                                oncoPrint() function from the BioConductor R
+                                package. Under the hood, the rendering is done using
+                                Plotly.js built upon D3. Plotly's interactivity allows
+                                the user to bind clicks and hovers to genetic events,
+                                allowing the user to create complex bioinformatic apps
+                                or workflows that rely on crossfiltering.
+                                """
+                            ),
+                            html.P(
+                                """
+                                Read more about the component here:
+                                https://github.com/plotly/react-oncoprint
+                                """
+                            )
+                        ])
+                    ),
+                    dcc.Tab(
+                        label='Data',
+                        value='data',
+                        children=html.Div(className='oncoprint-tab', children=[
+                            html.Div([
+                                html.Div(
+                                    className='oncoprint-option-name',
+                                    children='Select dataset'
+                                ),
+                                dcc.Dropdown(
+                                    id='oncoprint-dropdown',
+                                    className='oncoprint-select',
+                                    options=[
+                                        {
+                                            'label': '{}.json'.format(ds),
+                                            'value': ds
+                                        }
+                                        for ds in DATASETS
+                                    ],
+                                    value='cBioPortalData',
+                                ),
+                            ]),
+                            html.Hr(
+                                className='oncoprint-separator'
+                            ),
+                            html.Div([
+                                html.H4('Hover, click, or event data'),
```

Comment:
```suggestion
                                html.H4('Event metadata'),
```
seems to me, at least ('hover' and 'click' being event types, apparently)